net.sonic_pi.SonicPi.json

```diff
@@ -40,45 +40,56 @@
             "name": "erlang-otp",
             "config-opts": [
                 "--disable-pgo",
-	        "--without-hipe",
-	        "--without-javac", /* "--without-ssl", */
-	        "--without-erl_interface", /* "--without-asn1", */
-	        "--without-dialyzer",
-	        "--without-eldap",
-	        "--without-snmp",
-	        "--without-megaco",
-	        "--without-diameter",
-	        "--without-os_mon",
-	        "--without-mnesia",
-	        "--without-otp_mibs",
-	        "--without-common-test",
-	        "--without-debugger",
-	        "--without-edoc",
-	        "--without-erl_docgen",
-	        "--without-et",
-	        "--without-eunit",
-	        "--without-ftp",
-	        "--without-parsetools",
-	        "--without-reltool",
-	        "--without-sasl",
-	        "--without-ssh",
-	        "--without-syntax_tools",
-	        "--without-tftp",
-	        "--without-tools"
+                "--without-hipe",
+                "--without-javac", /* "--without-ssl", */
+                "--without-erl_interface", /* "--without-asn1", */
+                "--without-dialyzer",
+                "--without-eldap",
+                "--without-snmp",
+                "--without-megaco",
+                "--without-diameter",
+                "--without-os_mon",
+                "--without-mnesia",
+                "--without-otp_mibs",
+                "--without-common-test",
+                "--without-debugger",
+                "--without-edoc",
+                "--without-erl_docgen",
+                "--without-et",
+                "--without-eunit",
+                "--without-ftp",
+                "--without-reltool",
+                "--without-ssh",
+                "--without-syntax_tools",
+                "--without-tftp"
```

Comment:
`tools` and `parsetools` are needed to compile Elixir. `sasl` is required to compile the Sonic Pi Elixir app (tau).